harness/determined/util.py

```diff
@@ -225,3 +227,49 @@ def wrapper(*arg: List, **kwarg: Dict) -> Any:
         return cast(T, wrapper)
 
     return make_wrapper
+
+
+def safe_load_yaml_with_exceptions(yaml_file: Union[io.FileIO, IO[Any]]) -> Any:
```

Comment:
Can you put this in `common.util`?

Just to make the dependencies a little less circular.  It used to be that `determined_common` was a totally separate package that wasn't allowed to depend on `determined` proper at all.  That's not strictly the case but I think this should be in `common.util` anyway.

Reply:
Sure - should I merge after that, or ask Shiyuan for a review again?